pkgs/development/tools/gertty/default.nix

```diff
@@ -0,0 +1,48 @@
+{ python3, python3Packages, stdenv }:
+
+python3.pkgs.buildPythonPackage rec {
```

Comment:
If you only want to expose the `gertty` binary, please use `buildPythonPackage`, if you also want to expose gertty as python-library, please move the package to `python-modules`. (See 15.17.2.2.2 in the manual: https://nixos.org/nixpkgs/manual/)

Reply:
I choose the application, since it looks more what I'm trying to have here.